cdap-app-fabric/src/test/java/co/cask/cdap/AppWithSchedule.java

```diff
@@ -46,23 +55,59 @@ public void configure() {
       setDescription("Sample application");
       ObjectStores.createObjectStore(getConfigurer(), "input", String.class);
       ObjectStores.createObjectStore(getConfigurer(), "output", String.class);
-      addWorkflow(new SampleWorkflow());
+      AppConfig config = getConfig();
+      addFlow(new AllProgramsApp.NoOpFlow());
+      if (config.addWorkflow) {
+        addWorkflow(new SampleWorkflow());
+      }
 
       Map<String, String> scheduleProperties = Maps.newHashMap();
       scheduleProperties.put("oneKey", "oneValue");
       scheduleProperties.put("anotherKey", "anotherValue");
       scheduleProperties.put("someKey", "someValue");
 
-      scheduleWorkflow(Schedules.builder(SCHEDULE)
-                         .setDescription("Sample schedule")
-                         .createTimeSchedule("0/15 * * * * ?"),
-                       WORKFLOW_NAME,
-                       scheduleProperties);
+      if (config.addWorkflow && config.addSchedule1) {
+        scheduleWorkflow(Schedules.builder(SCHEDULE)
+                           .setDescription("Sample schedule")
+                           .createTimeSchedule("0/15 * * * * ?"),
+                         WORKFLOW_NAME,
+                         scheduleProperties);
+      }
+      if (config.addWorkflow && config.addSchedule2) {
+        scheduleWorkflow(Schedules.builder(SCHEDULE_2)
+                           .setDescription("Sample schedule")
+                           .createTimeSchedule("0/30 * * * * ?"),
+                         WORKFLOW_NAME,
+                         scheduleProperties);
+      }
     } catch (UnsupportedTypeException e) {
       throw Throwables.propagate(e);
     }
   }
 
+
+
+  /**
+   * Application Config Class to control schedule creation
+   */
+  public static class AppConfig extends Config {
+    private final boolean addWorkflow;
+    private final boolean addSchedule1;
+    private final boolean addSchedule2;
+
+    public AppConfig() {
+      this.addSchedule1 = true;
+      this.addSchedule2 = true;
```

Comment:
Hope this one doesn't break other unit tests. You will need to fix them there or just make this false by default.